src/replayToMovie.js

```diff
@@ -144,8 +150,15 @@ module.exports.runTestExport = async (outputPath, replay) => {
 module.exports.renderImages = async (replay, writer) => {
   const sprites = [];
   for (const frame of replay) {
-    for (let i = 0; i < frame.length; i++) {
-      const entry = frame[i];
+    // temporarily support both the new version of replay logs that contain
+    // sprites as well as envrionmental data, and the old version that contains
+    // just sprites
```

Comment:
Is the log format specification documented anywhere?  Will get really hard to change once we get start getting thousands of logs uploaded to S3.

(I guess we could make a breaking change, and no longer be able to process old logs that already have videos.)

Reply:
Good point. It isn't, but it should be.

I'm planning on implementing this transition as a breaking change; from the "draft" specification to the true v1. Future changes will definitely have to be handled better.